woocommerce/woocommerce
# Fix script version parameter for async chunks

### All Submissions:






### Changes proposed in this Pull Request:



Closes #33323.

This PR fixes the issue that async chunks are loaded without a version parameter since 6.5.

`AsyncChunkSrcVersionParameterPlugin`  is not compatible with webpack v5 because `JsonpMainTemplatePlugin.js` has been removed and we replaced `@automattic/mini-css-extract-plugin-with-rtl` with `mini-css-extract-plugin`.

I use [`__webpack_get_script_filename__`](https://webpack.js.org/api/module-variables/#__webpack_get_script_filename__-webpack-specific).  and [` __webpack_require__.miniCssF`](https://github.com/webpack-contrib/mini-css-extract-plugin/blob/752b913523077d74d575c777b2bcc3239b724688/src/index.js#L919) to do this instead.

### How to test the changes in this Pull Request:

1. Open dev tool
2. Go to Woocommerce > Home
3. Search "admin/chunks" in the network tab
4. Observe that all resources are requests with a version param.
![Screen Shot 2022-06-07 at 17 33 22](https://user-images.githubusercontent.com/4344253/172347718-e179b654-5581-4111-8cee-8de83893c16c.png)



### Other information:




### FOR PR REVIEWER ONLY:
